### PR TITLE
updating format for values

### PR DESCRIPTION
**Pull request by AmandaDoyle**
#492

### CODE REVIEW

File: facdb/sql/_create_facdb.sql [hunk 1]
```diff
@@ -2,7 +2,7 @@ DROP TABLE IF EXISTS facdb;
 SELECT
     TRIM(UPPER(facdb_base.facname)) as facname,
     facdb_address.addressnum,
-    facdb_address.streetname,
+    TRIM(regexp_replace(facdb_address.streetname, '\s+', ' ', 'g')) as streetname,
```
Comment (SPTKL) [inline facdb/sql/_create_facdb.sql:5]: this should be done within the facdb_address table 
https://github.com/NYCPlanning/db-facilities/blob/develop/facdb/sql/_create_facdb_address.sql
we want to make sure all the logic/cleaning happens in the same place

File: facdb/sql/_create_facdb.sql [hunk 2]
```diff
@@ -12,9 +12,9 @@ SELECT
     facdb_spatial.bbl::bigint as bbl,
     facdb_spatial.commboard as cd,
     facdb_spatial.nta,
-    facdb_spatial.council,
-    facdb_spatial.schooldist,
-    facdb_spatial.policeprct,
+    LPAD(facdb_spatial.council,2,'0') as council,
```
Comment (SPTKL) [inline facdb/sql/_create_facdb.sql:15]: this should happen in 
https://github.com/NYCPlanning/db-facilities/blob/5f88854eed5dbc2bea2ec47b277f804825eef6c8/facdb/sql/_create_facdb_spatial.sql#L41-L59
similar to bin formatting, all other districts formating should be done here ^

**Review by SPTKL — CHANGES_REQUESTED** [review]
move logic to their corresponding sql files, so that we can address issues more upstream

**Comment by mgraber** [comment]
> move logic to their corresponding sql files, so that we can address issues more upstream

The field `schooldist` has values with length less than 2 digits in 31 data sources:

- dca_operatingbusinesses
- dcla_culturalinstitutions
- dcp_colp
- dcp_pops
- dcp_sfpsd
- doe_lcgms
- dohmh_daycare
- dot_bridgehouses
- dot_mannedfacilities
- dot_pedplazas
- dot_publicparking
- dpr_parksproperties
- dsny_fooddrop
- dsny_garages
- dsny_leafdrop
- dsny_specialwastedrop
- dsny_textiledrop
- dycd_afterschoolprograms
- fdny_firehouses
- hhc_hospitals
- nycha_communitycenters
- nypl_libraries
- nysdec_lands
- nysdoh_healthfacilities
- nysed_activeinstitutions
- nysomh_mentalhealth
- nysparks_historicplaces
- sbs_workforce1
- usdot_airports
- usdot_ports
- usnps_parks

The field `council` has values with length less than 2 digits in 29 data sources:

- dca_operatingbusinesses
- dcla_culturalinstitutions
- dcp_colp
- dcp_pops
- dcp_sfpsd
- dohmh_daycare
- dot_bridgehouses
- dot_mannedfacilities
- dot_pedplazas
- dot_publicparking
- dpr_parksproperties
- dsny_fooddrop
- dsny_garages
- dsny_leafdrop
- dsny_specialwastedrop
- dsny_textiledrop
- dycd_afterschoolprograms
- fdny_firehouses
- hhc_hospitals
- nycha_communitycenters
- nypl_libraries
- nysdoh_healthfacilities
- nysed_activeinstitutions
- nysomh_mentalhealth
- nysparks_historicplaces
- sbs_workforce1
- usdot_airports
- usdot_ports
- usnps_parks

The field `policeprct` has values with length less than 3 digits in 37 data sources:

- bpl_libraries
- dca_operatingbusinesses
- dcla_culturalinstitutions
- dcp_colp
- dcp_pops
- dcp_sfpsd
- doe_busroutesgarages
- doe_lcgms
- dohmh_daycare
- dot_bridgehouses
- dot_mannedfacilities
- dot_pedplazas
- dot_publicparking
- dpr_parksproperties
- dsny_electronicsdrop
- dsny_fooddrop
- dsny_garages
- dsny_leafdrop
- dsny_specialwastedrop
- dsny_textiledrop
- dycd_afterschoolprograms
- fdny_firehouses
- hhc_hospitals
- nycha_communitycenters
- nycha_policeservice
- nypl_libraries
- nysdec_lands
- nysdoh_healthfacilities
- nysdoh_nursinghomes
- nysed_activeinstitutions
- nysomh_mentalhealth
- nysparks_historicplaces
- nysparks_parks
- sbs_workforce1
- usdot_airports
- usdot_ports
- usnps_parks

Is it worth padding in each of these individual sql files?

**Pull request merged**